src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -0,0 +1,86 @@
+                  {:title           (i18n/label :t/no-recent-transactions)
+                   :description     (i18n/label :t/make-one-it-is-easy-we-promise)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :saved       [quo/empty-state
+                  {:title           (i18n/label :t/no-saved-addresses)
+                   :description     (i18n/label :t/you-like-to-type-43-characters)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :contacts    [quo/empty-state
+                  {:title           (i18n/label :t/no-contacts)
+                   :description     (i18n/label :t/no-contacts-description)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]
+    :my-accounts [quo/empty-state
+                  {:title           (i18n/label :t/no-other-accounts)
+                   :description     (i18n/label :t/here-is-a-cat-in-a-box-instead)
+                   :placeholder?    true
+                   :container-style style/empty-container-style}]))
+
+(defn- view-internal
+  []
+  (let [margin-top   (safe-area/get-top)
+        timer        (atom nil)
+        valid-ens?   (reagent/atom false)
+        input-value  (atom "")
+        selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn [{:keys [theme]}]
+      [rn/scroll-view
+       {:content-container-style      (style/container margin-top)
+        :keyboard-should-persist-taps :never
+        :scroll-enabled               false}
+       [quo/page-nav
+        {:icon-name           :i/close
+         :on-press            #(rf/dispatch [:navigate-back])
+         :accessibility-label :top-bar
+         :right-side          :account-switcher}]
+       [quo/text-combinations
+        {:title                     (i18n/label :t/send-to)
+         :container-style           style/title-container
+         :title-accessibility-label :title-label}]
+       [quo/address-input
+        {:on-scan        #(js/alert "Not implemented yet")
+         :ens-regex      constants/regx-ens
+         :on-detect-ens  (fn [_]
+                           (reset! valid-ens? false)
+                           (when @timer
+                             (js/clearTimeout @timer))
+                           (reset! timer (js/setTimeout #(reset! valid-ens? true)
+                                                        2000)))
+         :on-change-text #(reset! input-value %)
+         :valid-ens?     @valid-ens?}]
+       [rn/view {:style (style/divider theme)}]
+       [quo/tabs
+        {:style            style/tabs
+         :content-style    style/tabs-content
+         :size             32
+         :default-active   @selected-tab
+         :data             tabs-data
+         :scrollable?      true
+         :scroll-on-press? true
+         :on-change        #(reset! selected-tab %)}]
```

Comment:
This function doesn't need to be recreated on every render because it depends only on the ratom, so it can be moved out of the renderer.